Documentation/rust/quick-start.rst

```diff
@@ -137,7 +137,7 @@ If you are using ``rustup``, its ``default`` profile already installs the tool,
 so you should be good to go. If you are using another profile, you can install
 the component manually::
 
-    rustup component add rustdoc
+    rustup component add rust-docs
```

Comment:
Rustdoc is part of the rustc component, which is mandatory for all profiles. The rust-docs component contains the generated html documentation for the sysroot crates.

```
$ cat $(rustc --print sysroot)/lib/rustlib/manifest-rustc-x86_64-unknown-linux-gnu | grep rustdoc
file:share/man/man1/rustdoc.1
file:bin/rustdoc

$ cat $(rustc --print sysroot)/lib/rustlib/manifest-rust-docs-x86_64-unknown-linux-gnu | head
file:share/doc/rust/html/SourceSerifPro-Bold.ttf.woff
file:share/doc/rust/html/guide-crates.html
file:share/doc/rust/html/main1.51.0.js
file:share/doc/rust/html/reference/tomorrow-night.css
file:share/doc/rust/html/reference/linkage.html
file:share/doc/rust/html/reference/attributes-redirect.html
file:share/doc/rust/html/reference/expressions.html
file:share/doc/rust/html/reference/influences.html
file:share/doc/rust/html/reference/fonts/open-sans-v17-all-charsets-800italic.woff2
file:share/doc/rust/html/reference/fonts/SOURCE-CODE-PRO-LICENSE.txt
```

Reply:
Oh, then we don't need this section entirely?

Reply:
It seems so.

Reply:
Fixed, thanks!